cgi-bin/LJ/S2/RecentPage.pm

```diff
@@ -90,6 +90,15 @@ sub RecentPage
         return 1;
     }
 
+    # libs for userpicselect
+    # if we're using the site skin, don't override the jquery-ui theme, as that's already included
+    my @iconbrowser_extra_stylesheet;
+    @iconbrowser_extra_stylesheet = ( 'stc/jquery/jquery.ui.theme.smoothness.css' )
+            unless $opts->{handle_with_siteviews_ref} && ${$opts->{handle_with_siteviews_ref}};
+
+    LJ::need_res( LJ::Talk::init_iconbrowser_js( 1, @iconbrowser_extra_stylesheet ) )
+        if $remote && $remote->can_use_userpic_select;
+
```

Comment:
I suspect this works just fine where it is, but for maintenance purposes, can you move this declaration up to be just after the related jquery includes, similar to the placement you used in DayPage.pm?

Duplicated code is my least favorite, but the pain is somewhat less if we can keep the related things together. :disappointed: